tuxemon/monster.py

```diff
@@ -691,7 +702,8 @@ def end_combat(self) -> None:
             self.status = []
 
     def speed_test(self, action: EnqueuedAction) -> int:
-        if action.technique.is_fast:
+        technique = cast(Technique, action.technique)
```

Comment:
Why is the cast needed? In case that is necessary, I would prefer an `assert`.

Reply:
```
class EnqueuedAction(NamedTuple):
    user: Union[Monster, NPC, None]
    technique: Union[Technique, Item]
    target: Monster
```

Reply:
An `assert` then, better than a cast in my opinion.

Reply:
Done.